runtime/pendulum/src/lib.rs

```diff
@@ -14,6 +14,7 @@ use crate::zenlink::*;
 use xcm::v1::MultiLocation;
 use zenlink_protocol::{AssetBalance, MultiAssetsHandler, PairInfo};
 
+pub use currency::CurrencyId as PendulumCurrencyId;
```

Comment:
This is not used.

Reply:
@TorstenStueber 
but if you want i create a pr just few sec ago to re export only for testing
https://github.com/pendulum-chain/pendulum/pull/200